### DISCUSSION TITLE
Getting Not Implemented Error() for 3d pose inference using MPI_INF_3DHP dataset

### DISCUSSION THREAD

**Discussion by smahmood00** [open-mmlab/mmpose#1548] [Q&A]
Hi, I am trying to do 2d to 3d pose lifting using the following line, where "pose_model" denotes the initialised model using config and checkpoint files, "pose_results"  is the estimated 2d pose keypoints of 2d pose inference. 
pose_results_3d =inference_pose_lifter_model(pose_model_3d,pose_results, dataset='MPI-INF-3DHP')

However, I am getting Not Implemented Error().

Your help and suggestion will be highly appreciated. Thanks in advance.

**Comment by ly015** [comment]
You may need to set the `dataset_info` argument. Please refer to https://github.com/open-mmlab/mmpose/blob/master/demo/body3d_two_stage_video_demo.py#L311-L318

**Reply by smahmood00** [reply]
Thanks for your prompt suggestion! Now I get the following error..
Traceback (most recent call last):
  File "Human_3D.py", line 43, in <module>
    list_dict=inference_pose_lifter_model(pose_model_3d,pose_results, dataset='MPI-INF-3DHP',dataset_info=dataset_info)
  File "/research/d4/spc/smahmood0/miniconda3/envs/openmmlab/lib/python3.8/site-packages/mmpose/apis/inference_3d.py", line 261, in inference_pose_lifter_model
    flip_pairs = dataset_info.flip_pairs
  File "/research/d4/spc/smahmood0/miniconda3/envs/openmmlab/lib/python3.8/site-packages/mmcv/utils/config.py", line 50, in __getattr__
    raise ex
AttributeError: 'ConfigDict' object has no attribute 'flip_pairs'

**Reply by ly015** [reply]
Did you pass a config object to the `dataset_info` argument? Actually, it should be a `DatasetInfo` instance built from the config. Please check https://github.com/open-mmlab/mmpose/blob/master/demo/body3d_two_stage_video_demo.py#L318

**Comment by smahmood00** [comment]
Yes I did that:
# initialize detector
det_model = init_detector(det_config, det_checkpoint)
# initialize pose model
pose_model = init_pose_model(pose_config, pose_checkpoint)

dataset_info=pose_model.cfg.data['test'].get('dataset_info', None)
